cqf-fhir-cr/src/test/java/org/opencds/cqf/fhir/cr/measure/r4/MeasureEvaluatedResourcesTest.java

```diff
@@ -0,0 +1,121 @@
+package org.opencds.cqf.fhir.cr.measure.r4;
+
+import static org.opencds.cqf.fhir.test.Resources.getResourcePath;
+
+import ca.uhn.fhir.context.FhirContext;
+import java.nio.file.Paths;
+import org.hl7.fhir.r4.model.DateTimeType;
+import org.hl7.fhir.r4.model.Period;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.opencds.cqf.fhir.api.Repository;
+import org.opencds.cqf.fhir.cr.measure.r4.Measure.Given;
+import org.opencds.cqf.fhir.cr.measure.r4.utils.TestDataGenerator;
+import org.opencds.cqf.fhir.utility.repository.ig.IgRepository;
+
+/**
+ * When Measure Evaluation produces a Measure Report of type 'Individual', a evaluatedResources field
+ * should populate reference values of resources interacted with to produce a MeasureScore or SDE.
+ * This should apply for:
+ * * supplementalDataElements (SDE)
+ * * populations (ex: initial-population), which include stratifiers
+ * No duplicate references in evaluated resources
+ */
+public class MeasureEvaluatedResourcesTest {
+    private static final String CLASS_PATH = "org/opencds/cqf/fhir/cr/measure/r4";
+    private static final Repository repository = new IgRepository(
+            FhirContext.forR4Cached(),
+            Paths.get(getResourcePath(MeasureStratifierTest.class) + "/" + CLASS_PATH + "/" + "MeasureTest"));
+    protected Given given = Measure.given().repository(repository);
+    private static final TestDataGenerator testDataGenerator = new TestDataGenerator(repository);
+
+    @BeforeAll
+    static void init() {
+        Period period = new Period();
+        period.setStartElement(new DateTimeType("2024-01-01T01:00:00Z"));
```

Comment:
I really think we should make period start and end into flexpoints for these tests so that we can more easily find and fix any future timezone bugs.